src/azure-cli/azure/cli/command_modules/keyvault/_transformers.py

```diff
@@ -3,6 +3,35 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from collections.abc import Iterable
+
+
+def multi_transformers(*transformers):
+    def _multi_transformers(output):
+        for t in transformers:
+            output = t(output)
+        return output
+    return _multi_transformers
+
 
 def filter_out_managed_resources(output):
     return [_ for _ in output if not getattr(_, 'managed')] if output else output
+
+
+def _extract_subresource_name_from_single_output(output, id_parameter):
+    if not getattr(output, id_parameter):
+        resource_name = None
+    else:
+        items = getattr(output, id_parameter).split('/')
+        resource_name = items[4] if len(items) > 4 else None
```

Comment:
Can we use `parse_resource_id` here?

Reply:
> `parse_resource_id`

@fengzhou-msft Good idea! I will use `parse_resource_id`.